doc/RDF_tools.ipynb

```diff
@@ -0,0 +1,732 @@
+      "Property: https://g-node.org/projects/odml-rdf#fadffec7-6b23-454e-bfd1-9d5884802abb\n",
+      "Section: https://g-node.org/projects/odml-rdf#bbd44815-5016-49e0-9f4b-5b83778d00de\n",
+      "\n",
+      "SELECT * WHERE {\n",
+      "?d odml:hasSection ?s .\n",
+      "?s rdf:type odml:Section .\n",
+      "?s odml:hasName \"Recording-2013-02-08-ak\" .\n",
+      "?s odml:hasType \"Recording\" .\n",
+      "}\n",
+      "Document: https://g-node.org/projects/odml-rdf#537c6cc8-7dfe-4d53-a111-24b3ce0f3c1a\n",
+      "Section: https://g-node.org/projects/odml-rdf#346773f2-abee-4892-b052-840ddcff35ee\n",
+      "\n",
+      "SELECT * WHERE {\n",
+      "?s odml:hasProperty ?p .\n",
+      "?p rdf:type odml:Property .\n",
+      "?p odml:hasName \"Date\" .\n",
+      "}\n",
+      "Property: https://g-node.org/projects/odml-rdf#1d6db4ce-87f3-4e9c-b221-e76ba05b2759\n",
+      "Section: https://g-node.org/projects/odml-rdf#782bd29d-e4b0-4c14-a417-1772a4851ffd\n",
+      "Property: https://g-node.org/projects/odml-rdf#fadffec7-6b23-454e-bfd1-9d5884802abb\n",
+      "Section: https://g-node.org/projects/odml-rdf#bbd44815-5016-49e0-9f4b-5b83778d00de\n",
+      "Property: https://g-node.org/projects/odml-rdf#f1699eb6-4cab-4dd0-9327-120eab2089ae\n",
+      "Section: https://g-node.org/projects/odml-rdf#5365f7e5-603c-4154-a5ea-33bb1a07a956\n",
+      "Property: https://g-node.org/projects/odml-rdf#138f08f7-23c7-4722-8577-85a6fa633ae1\n",
+      "Section: https://g-node.org/projects/odml-rdf#346773f2-abee-4892-b052-840ddcff35ee\n",
+      "\n",
+      "SELECT * WHERE {\n",
+      "?d odml:hasSection ?s .\n",
+      "?s rdf:type odml:Section .\n",
+      "?s odml:hasName \"Recording-2013-02-08-ak\" .\n",
+      "}\n",
+      "Document: https://g-node.org/projects/odml-rdf#537c6cc8-7dfe-4d53-a111-24b3ce0f3c1a\n",
+      "Section: https://g-node.org/projects/odml-rdf#346773f2-abee-4892-b052-840ddcff35ee\n",
+      "\n",
+      "SELECT * WHERE {\n",
+      "?d odml:hasSection ?s .\n",
+      "?s rdf:type odml:Section .\n",
+      "?s odml:hasType \"Recording\" .\n",
+      "}\n",
+      "Document: https://g-node.org/projects/odml-rdf#cc66e78a-3742-490a-9fdb-1c66761d7652\n",
+      "Section: https://g-node.org/projects/odml-rdf#5365f7e5-603c-4154-a5ea-33bb1a07a956\n",
+      "Document: https://g-node.org/projects/odml-rdf#537c6cc8-7dfe-4d53-a111-24b3ce0f3c1a\n",
+      "Section: https://g-node.org/projects/odml-rdf#346773f2-abee-4892-b052-840ddcff35ee\n",
+      "Document: https://g-node.org/projects/odml-rdf#cd24b60f-1d5e-4040-9881-5e5a597baef7\n",
+      "Section: https://g-node.org/projects/odml-rdf#782bd29d-e4b0-4c14-a417-1772a4851ffd\n",
+      "Document: https://g-node.org/projects/odml-rdf#24066355-1ee8-4eb5-a715-96bbb6231cd5\n",
+      "Section: https://g-node.org/projects/odml-rdf#bbd44815-5016-49e0-9f4b-5b83778d00de\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from odml.tools.fuzzy_finder import FuzzyFinder\n",
```

Comment:
I tested the notebook with both python2 and python3. Works like a charm for python3, with python2 at this point I get the following error: `ImportError: cannot import name ABC` from `query_creator.py` line 2.

Reply:
Changed to inheritance to `__metaclass__ = ABCMeta`.